bld/CLMBuildNamelist.pm

```diff
@@ -2343,6 +2354,12 @@ sub setup_logic_do_transient_pfts {
    }
 }
 
+   if (&value_is_true($nl->get_value($var))) {
```

Comment:
It looks like now this block is too low - after the end of the function. It should be before the ending curly brace that is currently on line 2355.

Reply:
Oops, thanks for catching that! It's prob. clear from this back and forth that I'm not experienced with this scripting language (what language is it btw?). Though my detail oriented self should have noticed that ending curly brace...